tests/cases/fourslash/noErrorsAfterCompletionsRequestWithinGenericFunction1.ts

```diff
@@ -0,0 +1,9 @@
+///<reference path="fourslash.ts"/>
+// @strict: true
+////
+//// declare function func<T extends { foo: 1 }>(arg: T): void;
+//// func({ foo: 1, bar/*1*/: 1 });
+
+goTo.marker("1");
+verify.completions({ exact: undefined });
+verify.noErrors();
```

Comment:
this was reporting excess property check - I still think that perhaps it **shouldn't** cause applicability error in this case when requesting completions but that's a separate problem